plugins/catalog-graph/src/components/EntityRelationsGraph/types.ts

```diff
@@ -108,3 +108,13 @@ export enum Direction {
    */
   RIGHT_LEFT = 'RL',
 }
+
+/**
+ *  Graph curve factory
+ *
+ * @public
+ *
+ * @see {@link @types/d3-shape/index.d.ts#curveMonotoneX}
+ * @see {@link @types/d3-shape/index.d.ts#curveStepBefore}
+ */
+export type Curve = 'curveStepBefore' | 'curveMonotoneX';
```

Comment:
This can also be inlined. Otherwise let's not export it all the way out into the `api-report.md`

Reply:
@benjdlambert Done! Please let me know if everything is fine. Thanks!